ietf/group/tests_info.py

```diff
@@ -950,6 +950,31 @@ def test_edit_description_field(self):
         group = Group.objects.get(pk=group.pk)  # refresh
         self.assertEqual(group.description, 'Updated description')
 
+    def test_edit_parent_field(self):
+        group = GroupFactory(acronym='mars', parent=Group.objects.get(acronym='farfut'))
+        RoleFactory(group=group, name_id='chair')
+        url = urlreverse('ietf.group.views.edit', kwargs={'acronym': group.acronym, 'action': 'edit', 'field': 'parent'})
+
+        # parent is not shown to group chair
+        login_testing_unauthorized(self, 'mars-chair', url)
+        r = self.client.get(url)
+        self.assertEqual(r.status_code, 302)
+        self.assertEqual(len(r.content), 0)
```

Comment:
It's not clear what the intent of this test block is? The view that would only edit the parent field should probably just refuse entry?

What about testing the form that allows editing the whole group (when you don't pass in field to the view). - I would expect a test that shows that the parent field isn't present on that form.

Reply:
For the chair, /edit/parent/ behaves exactly like /edit/xyzzy/ - it renders an edit page with no editable fields, just a Submit button and a Back button, which both lead back to the about page.

However, I bolluxed the user creation, and what's actually going on there is a redirection to the login page. 🤦 

But yeah, I can test the whole group form.